README.md

```diff
@@ -8,6 +8,10 @@ A [Docker](https://www.docker.com/)-based installer and runtime for the [Symfony
 2. Open `https://localhost` in your favorite web browser and [accept the auto-generated TLS certificate](https://stackoverflow.com/a/15076602/1352334)
 3. **Enjoy!**
 
+### Note for linux users
```

Comment:
May I suggest:

    ### Editing `vendor/` and `var/` directories on Linux

Can you also add this section just before the "Credits" one?

Reply:
Well the problem here is that is not only the vendor and var directory is not writable, but even the src and config because the very first time you execute `docker-compose up` these files doesn't exist, thus are created when containers are started (by the entrypoint).

So to reflect this the title could become
```
### Editing source files on Linux
```

Ok so I'll add the section as the last h3 of the Debugging section, right after troubleshooting.

Reply:
The command edits all the files of the container ?
If so, in which case this would agree more :
```
### Editing permissions on Linux
```
WDYT ?

Reply:
Yes, this command sets your user as owner for all the files through the container.
This is useful only for dev - when you use mounted volumes - because you change project files in your host machine as well;
in production (when you shouldn't use volumes) it's not really useful because you can handle permission issues directly in dockerfile, like you already do [here](https://github.com/dunglas/symfony-docker/blob/master/Dockerfile#L87)

However `Editing permissions on Linux` as a section title is fine for me.

Reply:
Go for `Editing Permissions on Linux` (in title case please). Thanks!